ios/FluentUI/Extensions/EnvironmentValues+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+import SwiftUI
+
+struct SafeAreaInsetsKey: EnvironmentKey {
+    static var defaultValue: EdgeInsets {
+        return UIApplication.shared.keyWindow?.safeAreaInsets.swiftUIInsets ?? EdgeInsets()
+    }
+}
+
+extension EnvironmentValues {
+  var swiftUIInsets: EdgeInsets {
+    get { self[SafeAreaInsetsKey.self] }
+    set { self[SafeAreaInsetsKey.self] = newValue }
+  }
+}
+
+private extension UIEdgeInsets {
+    var swiftUIInsets: EdgeInsets {
+        EdgeInsets(top: top, leading: left, bottom: bottom, trailing: right)
```

Comment:
It's okay if this is a short-term patch, but long-term it should be pointed out that this is could cause issues in RTL languages. Leading should map to `right` and trailing to `left` in those cases.

Might be worth at least commenting to note this deficiency if we don't want to fix it right now.